### PR TITLE
Bulk updates to homepage

### PR DESCRIPTION
**Pull request by walkerford**
Bulk updates to homepage.

### CODE REVIEW

**Comment by walkerford** [comment]
Remaining todos:

- [ ] icon attribution

File: content/en/curriculum/getting-started/objectives.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Sections
+
+The Launchpad curriculum is split into these topic areas:
+
+- Web3 Basics
+- The Protocol Labs Network
+- IPFS -- voluntary decentralized file storage
+- Filecoin -- incentivized decentralized file storage
+- IPLD -- universal method to represent data
```
Comment (momack2) [inline content/en/curriculum/getting-started/objectives.md:32]:
```suggestion
- IPLD -- data-model for content addressed data
```

File: content/en/curriculum/getting-started/objectives.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- libp2p -- universal peer-to-peer networking
```
Comment (momack2) [inline content/en/curriculum/getting-started/objectives.md:33]:
```suggestion
- libp2p -- universal peer-to-peer networking library
```

File: content/en/curriculum/getting-started/objectives.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+disappear when you select "deep".
+
+![Levels](levels.png)
+
+## Study approach
+
+1. Read through the material.
+2. Watch the reference videos either as you go or at the end.
+3. Once you've read and watched, you are then ready to view on of our recorded
+   sessions. In the past we would host live sessions with engineers who would
```
Comment (momack2) [inline content/en/curriculum/getting-started/objectives.md:48]:
```suggestion
   Q&A sessions. In the past we would host live sessions with engineers who would
```

File: content/en/curriculum/getting-started/objectives.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+   talk about the technologies and answer questions. We have some of those
+   recorded and linked for you to watch.
+4. At this point you have completed everything for the course. You next step
```
Comment (momack2) [inline content/en/curriculum/getting-started/objectives.md:51]:
```suggestion
4. At this point you have completed everything for the async course. Your next step
```

File: content/en/curriculum/getting-started/objectives.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+   can be to reach out in Filecoin Slack if you have questions. Go to github
+   and check out our repos. There are issues tagged as good starting tasks, if
```
Comment (momack2) [inline content/en/curriculum/getting-started/objectives.md:53]:
```suggestion
   and check out the associated repos. There are issues tagged as good starting tasks, if
```

File: content/en/curriculum/web3/blockchains/index.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+Consensus is the process whereby nodes that don't know each other can come into
+agreement regarding the validity and ordering of transactions. Decentralized
+consensus is difficult because it has to work even when there are some number of
+hostile actors who are deliberately trying to game the system. In the case of
+most blockchains, as long as >50% of the actors are good actors, who play by the
+rules, the system will function properly.
+
+How do blockchains ensure they retain a majority of good actors?
+
+There is no guarantee, but there ways to encourage success. Incentivizes are
```
Comment (momack2) [inline content/en/curriculum/web3/blockchains/index.md:78]:
```suggestion
There is no guarantee, but there ways to encourage success. Incentives are
```

File: content/en/curriculum/web3/blockchains/index.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+
+![Blockchain visualization](blockchain1.png) ![Blockchain
+visualization](blockchain2.png)
+
+## Consensus
+
+Consensus is the process whereby nodes that don't know each other can come into
+agreement regarding the validity and ordering of transactions. Decentralized
+consensus is difficult because it has to work even when there are some number of
+hostile actors who are deliberately trying to game the system. In the case of
```
Comment (momack2) [inline content/en/curriculum/web3/blockchains/index.md:72]:
```suggestion
hostile actors who are deliberately trying to game the system (even at their own cost). In the case of
```

File: content/en/curriculum/web3/blockchains/index.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+most blockchains, as long as >50% of the actors are good actors, who play by the
```
Comment (momack2) [inline content/en/curriculum/web3/blockchains/index.md:73]:
```suggestion
most blockchains, as long as >50% of the actors are rational actors, who play by the
```

File: content/en/curriculum/web3/blockchains/index.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+constructed in such a way that playing by the rules will yield a greater result
+than breaking the rules. An actor who assaults the system may be charged a
+significant fee taken from their bonded collateral. Ultimately, an actor who
+damages the network permanently will end up with a network that is not worth
+anything. These kinds of game-theoretic incentives have worked so far. Although
+there have been hacks and other minor exploits, the systems have proven to be
+stable and trustworthy. The hacks and exploits will reduce in time as these
+systems mature.
+
+How does consensus work?
```
Comment (momack2) [inline content/en/curriculum/web3/blockchains/index.md:88]:
```suggestion
### How does consensus work?
```

File: content/en/curriculum/web3/cryptography/index.md [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+maintain sovereignty of and freedom on the internet.
+
+“We cannot expect governments, corporations, or other large, faceless
+organizations to grant us privacy out of their beneficence. [...] We must defend
+our own privacy if we expect to have any. We must come together and create
+systems which allow anonymous transactions to take place.” (A Cypherpunk’s
+Manifesto)
+
+A decentralized digital currency must not have a central authority that could be
+shut down. But without a the central authority, the problem arises of how to
```
Comment (momack2) [inline content/en/curriculum/web3/cryptography/index.md:152]:
```suggestion
shut down. But without a central authority, the problem arises of how to
```

File: content/en/curriculum/web3/objectives/index.md [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+category: lecture
+weight: 5
+level:
+  - shallow
+  - deep
+---
+
+## Overview
+
+The internet is a result humanity's drive to develop more efficient ways of
```
Comment (momack2) [inline content/en/curriculum/web3/objectives/index.md:17]:
```suggestion
The internet is a result of humanity's drive to develop more efficient ways of
```

File: content/en/curriculum/web3/scaling/index.md [hunk 12]
```diff
@@ -0,0 +1,265 @@
...
+- Zero-knowledge (Zk) Rollups: runs computation off-chain and submits a validity
+  proof to the chain
+
+“In general, my own view is that in the short term, optimistic rollups are
+likely to win out for general-purpose EVM computation and ZK rollups are likely
+to win out for simple payments, exchange and other application-specific use
+cases, but in the medium to long term ZK rollups will win out in all use cases
+as ZK-SNARK technology improves.”
+
+Vitalik Buterin, An Incomplete Guide to Rollups
```
Comment (momack2) [inline content/en/curriculum/web3/scaling/index.md:73]:
```suggestion
-- Vitalik Buterin, An Incomplete Guide to Rollups
```

File: content/en/curriculum/web3/storage/index.md [hunk 13]
```diff
@@ -0,0 +1,112 @@
...
+data sets that would prefer to optimize on costs, availability or other
+parameters.
+
+---
+
+Filecoin storage estimates from
+[this](https://www.coingecko.com/research/publications/centralized-decentralized-storage-cost)
+Coingecko report, updated May 2023.
+
+The Arweave and Storj from each of their respective websites.
```
Comment (momack2) [inline content/en/curriculum/web3/storage/index.md:112]: the network size and cost estimate?

File: content/en/curriculum/web3/storage/index.md [hunk 14]
```diff
@@ -0,0 +1,112 @@
...
+promise that the user will not have to worry about renegotiating deals in the
+future. Data is stored in a sharded sort of way, with each node storing only a
+small portion of the networks data, but all data being backed up across hundreds
+of other nodes. Statistical checks on data consistency are performed on a
+continual basis. Nodes that cannot prove they have their portion of the data
+will be punished through fines (loss of collateral) and missing data will be
+restored from and reassigned to other nodes.
+
+Arweave priorities ease of use over configurability, which makes it will suited
+for simple use cases nad perhaps less well suited for larger or more complex
```
Comment (momack2) [inline content/en/curriculum/web3/storage/index.md:102]: nad --> and

File: content/en/curriculum/web3/storage/index.md [hunk 15]
```diff
@@ -0,0 +1,112 @@
...
+
+Token: FIL
+
+[Filecoin website](https://filecoin.io/store/#intro)
+
+- Total network capacity: ~13,000 PB
+- Total data stored: ~1,300 PB
+- Storage cost: USD$0-0.19 per TB-month (varies per storage provider and file size)
+
+Filecoin is the largest decentralized storage marketplace by several orders of magnitude. It is based off of the popular IPFS protocol, which is the voluntary storage network that stores many of the NFTs on Ethereum, but adds an incentivization layer that guarantees storage over the duration of a deal period. Instead of relying on statistical consistency checks, Filecoin uses a sophisticated cryptographic algorithm to prove that all data is being accurately stored across all storage providers on a daily basis. Such checks provide the most reliable guarantee that data has not gone stale. Filecoin is a protocol that strives to provide the maximum flexibility in terms of storage parameters. Users can configure the deal duration, redundancy, storage provider location, read-performance, and other parameters. Filecoin relies on an ecosystem of interface providers to offer users
```
Comment (momack2) [inline content/en/curriculum/web3/storage/index.md:48]: ends abruptly? maybe "to offer users more choice and configurability"?

File: layouts/index.html [hunk 16]
```diff
@@ -1,3 +1,1160 @@
...
+                      transform: translate(0px, 0%);
+                    "
+                  >
+                    learning!
+                  </div>
+                </div>
+              </div>
+              <div class="text-lg">
+                Protocol Labs starts & supports breakthroughs in computing to
+                push humanity forward.
```
Comment (momack2) [inline layouts/index.html:557]:
```suggestion
                push humanity forward. Launchpad can help with:
```

File: layouts/index.html [hunk 17]
```diff
@@ -1,3 +1,1160 @@
...
+              <div
+                class="flex flex-col lg:flex-row lg:justify-evenly gap-10 mt-10"
+              >
+                <div class="flex flex-row">
+                  <img class="lp-icon" src="icons/hello.png" alt="" />
+                  <h3 class="item">
+                    <span class="text-color-light-blue">Onboarding</span> and
+                    connecting
+                    <span class="text-color-light-blue">new hires</span>
+                    from PLN teams to Web3 & the PL Network
```
Comment (momack2) [inline layouts/index.html:568]:
```suggestion
                    to Web3 & the PL Network
```

File: layouts/index.html [hunk 18]
```diff
@@ -1,3 +1,1160 @@
...
+                    Providing the learning and technical knowledge to quickly
+                    prepare employees
+                    <span class="text-color-light-blue">to succeed</span> in
+                    their their roles in Web3
+                  </h3>
+                </div>
+                <div class="flex flex-row">
+                  <img class="lp-icon" src="icons/shuttle.png" alt="" />
+                  <h3 class="item">
+                    Accelerating talent into becoming
```
Comment (momack2) [inline layouts/index.html:584]:
```suggestion
                    Accelerating new talent to become
```

File: layouts/index.html [hunk 19]
```diff
@@ -1,3 +1,1160 @@
...
+          <div class="container">
+            <div class="home-network_component">
+              <div class="flex flex-col items-center">
+                <h2 class="heading-50 text-color-grey">
+                  A Training Program for
+                  <span class="text-color-dark-blue"> You! </span>
+                </h2>
+                <div class="mt-10 mb-5 text-18 text-color-grey">
+                  Click on the video to learn more about the Launchpad journey!
+                </div>
```
Comment (momack2) [inline layouts/index.html:605]:
```suggestion
```

Comment (momack2) [reply layouts/index.html:605]: This is duplicative. Also - we should never say "click x" on a website (looks bad)

File: layouts/index.html [hunk 20]
```diff
@@ -1,3 +1,1160 @@
...
+                    >
+                    across the PLN, Launchpad has served as the catalyst for the
+                    growth and development of talented
+                    <span class="text-22 text-color-dark-blue">developers</span
+                    >, <span> marketing managers </span>,
+                    <span class="text-22 text-color-dark-blue">recruiters</span
+                    >, and many more professionals across the ecosystem. With
+                    our curriculum, you will be able to learn about Web3 at
+                    <span class="text-22 text-color-dark-blue"
+                      >your own pace!</span
```
Comment (momack2) [inline layouts/index.html:645]:
```suggestion
                    Launchpad is a hub for
                    <span class="text-22 text-color-dark-blue">
                      web3 learning</span
                    >
                    and onboarding within the Protocol Labs Network, having
                    welcomed over
                    <span class="text-22 text-color-dark-blue"
                      >200 learners all over the world!</span
                    >
                    From newcomers to seasoned professionals, Launchpad provides essential knowledge and
                    support for individuals venturing into the exciting world of web3.
                  </div>

                  <div class="text-color-grey">
                    From guides on how to set up your first IPFS or Filecoin
                    node, to fostering
                    <span class="text-22 text-color-dark-blue"
                      >collaborative connections</span
                    >
                    across the PLN, Launchpad has served as the catalyst for the
                    growth and development of talented
                    <span class="text-22 text-color-dark-blue">developers</span
                    >, <span> marketing managers</span>,
                    <span class="text-22 text-color-dark-blue">recruiters</span
                    >, and many more across the ecosystem. The Launchpad curriculum helps anyone learn about Web3 at
                    <span class="text-22 text-color-dark-blue"
                      >your own pace!</span
```

File: layouts/index.html [hunk 21]
```diff
@@ -1,3 +1,1160 @@
...
+                      menu to begin your learning experience!"
+                    </div>
+                  </div>
+                  <div class="flex flex-row">
+                    <img class="lp-icon" src="icons/hello.png" alt="" />
+                    <div>
+                      To get in touch with us at Protocol Labs, Join us on
+                      <a href="https://filecoin.io/slack">Filecoin Slack</a>
+                      or fill the form and join our waitlist.
+                    </div>
```
Comment (momack2) [inline layouts/index.html:1124]:
```suggestion
                  The Launchpad curriculum covers topics like:
                  <span class="text-color-light-blue">blockchains</span>,
                  peer-to-peer communication, <span>content addressing</span>,
                  and
                  <span class="text-color-light-blue"
                    >decentralized file storage</span>.
                </div>
                <div>
                  Launchpad has proven to be a valuable resource to
                  <span class="text-color-light-blue">over 270 participants </span>
                  from 20 diverse organizations, with both synchronous and async cohorts of
                  learners diving into the curriculum together.
                </div>
                <hr class="h-px bg-gray-200 border-0 dark:bg-gray-700" />
                <div>
                  Launchpad
                  <span class="text-color-light-blue"
                    >live cohorts are currently paused</span
                  >. However the curriculum and resources are available to use for
                  <span class="text-color-light-blue"
                    >self-guided exploration</span
                  >.
                </div>
                <div class="flex flex-col gap-10 ml-5 mt-3">
                  <div class="flex flex-row">
                    <img class="lp-icon" src="icons/shuttle.png" alt="" />
                    <div class="">
                      <a href="curriculum/">Explore the Curriculum!</a> to begin your learning experience!
                    </div>
                  </div>
                  <div class="flex flex-row">
                    <img class="lp-icon" src="icons/hello.png" alt="" />
                    <div>
                      Get in touch with us on
                      <a href="https://filecoin.io/slack">Filecoin Slack</a>
                      or fill out the form to join our waitlist.
                    </div>
```

File: content/en/curriculum/web3/storage/index.md (hunk 14 above)
Comment (momack2) [reply content/en/curriculum/web3/storage/index.md:102]:
```suggestion
for simple use cases and perhaps less well suited for larger or more complex
```

**Review by momack2 — APPROVED** [review]

File: layouts/index.html [hunk 22]
```diff
@@ -1,3 +1,1143 @@
...
+                </div>
+                <div class="flex flex-row">
+                  <img class="lp-icon" src="icons/shuttle.png" alt="" />
+                  <h3 class="item">
+                    Accelerating new talent to become
+                    <span class="text-color-light-blue">connected</span>,
+                    <span class="text-color-light-blue">effective</span>, and
+                    <span class="text-color-light-blue">productive</span>
+                    contributors
+                  </h3>
```
Comment (momack2) [inline layouts/index.html:588]:
```suggestion
              <div
                class="flex flex-col lg:flex-row lg:justify-evenly gap-10 mt-10"
              >
                <div class="flex flex-row">
                  <img class="lp-icon" src="icons/hello.png" alt="" />
                  <h3 class="item">
                    <span class="text-color-light-blue">Onboard</span> and
                    connect
                    <span class="text-color-light-blue">new hires</span>
                    to Web3 & the PL Network
                  </h3>
                </div>

                <div class="flex flex-row">
                  <img class="lp-icon" src="icons/bulb.png" alt="" />
                  <h3 class="item">
                    Gain the technical knowledge and network know-how for new hires to
                    <span class="text-color-light-blue">succeed</span> in
                    web3
                  </h3>
                </div>
                <div class="flex flex-row">
                  <img class="lp-icon" src="icons/shuttle.png" alt="" />
                  <h3 class="item">
                    Accelerate new talent to become
                    <span class="text-color-light-blue">connected</span>,
                    <span class="text-color-light-blue">effective</span>, and
                    <span class="text-color-light-blue">productive</span>
                    contributors within web3
                  </h3>
```

File: content/en/curriculum/web3/storage/index.md (hunk 15 above)
Comment (walkerford) [reply content/en/curriculum/web3/storage/index.md]: Fixed

**Pull request merged**